docs/configurations.md

```diff
@@ -0,0 +1,266 @@
+|----------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|
+| scalar.db.metadata.cache_expiration_time_secs                  | ScalarDB has a metadata cache to reduce the number of requests to the database. This setting specifies the expiration time of the cache in seconds.                                                           | -1 (no expiration) |
+| scalar.db.active_transaction_management.expiration_time_millis | ScalarDB maintains ongoing transactions, which can be resumed using a transaction ID. This setting specifies the expiration time of this transaction management feature in milliseconds.                      | -1 (no expiration) |
+| scalar.db.default_namespace_name                               | The given namespace name will be used by operations that do not already specify a namespace. If you would like to use this setting with ScalarDB server, it needs to be set on the client-side configuration. |                    |
+
+## Configuration examples
+
+This section shows several configuration examples.
+
+### Example 1
+
+```
+[App (ScalarDB Library with Consensus Commit)] ---> [Underlying storage/database]
+```
+
+In this setting, App (ScalarDB Library with Consensus Commit) connects to the underlying storage/database (in this case Cassandra) directly.
+Note that this setting exists only for development purposes and is not recommended for production use.
+This is because the App needs to implement [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface to take transactionally-consistent backups for ScalarDB, which requires an extra burden for users.
+
+In this case, an example of configurations in App are as follows:
+```properties
+# Transaction manager implementation
+scalar.db.transaction_manager=consensus-commit
+
+# Storage implementation
+scalar.db.storage=cassandra
+
+# Comma separated contact points
+scalar.db.contact_points=<cassandra host>
+
+# Credential information to access the database
+scalar.db.username=<username>
+scalar.db.password=<password>
+```
+
+### Example 2
+
+```
+[App (ScalarDB Library with gRPC)] ---> [ScalarDB Server (ScalarDB Library with Consensus Commit)] ---> [Underlying storage/database]
+```
+
+In this setting, App (ScalarDB Library with gRPC) connects to an underlying storage/database through ScalarDB Server.
+This setting is recommended for production use.
+This is because ScalarDB Server implements [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface, which enables you to take transactionally-consistent backups for ScalarDB by pausing the ScalarDB Server.
```

Comment:
```suggestion
In this configuration, the app (ScalarDB Library with gRPC) connects to an underlying storage/database through ScalarDB Server.
This configuration is recommended for production use because ScalarDB Server implements the [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface, which enables you to take transactionally consistent backups for ScalarDB by pausing ScalarDB Server.
```